openapi/propolis-server.json

```diff
@@ -662,6 +662,13 @@
             "additionalProperties": {
               "$ref": "#/components/schemas/StorageDeviceV0"
             }
+          },
+          "boot_order": {
+            "nullable": true,
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/BootDeclaration"
```

Comment:
@gjcolombo my current thinking is that we want to keep this as an array of structs for forward compatibility reasons, though that probably matters a lot less between Nexus and Propolis. wdyt? should it just be an array of string (or single string for the time being)

Reply:
went with pretty much this, couldn't think of any downsides :) https://github.com/oxidecomputer/propolis/pull/756/commits/6a386b0806a9b5dcba6924619ac1afc541bc8715